jumpscale/packages/tfgrid_solutions/chats/cryptpad_deploy.py

```diff
@@ -81,6 +82,9 @@ def select_pool(self):
     @chatflow_step(title="Network")
     def cryptpad_network(self):
         self.network_view = deployer.select_network(self)
+
+    @chatflow_step(title="")
+    def container_node_id(self):
```

Comment:
we can merge this too with container ip with domain, under title `Choose your domain`